### ISSUE TITLE
Missing purified[i] variable in src/Adversarial_examples.ipynb

### ISSUE THREAD

**Issue by haloha123** [ankile/Adversarial-Diffusion#1]
Thank you for the work!

I found an missing term `purified` in your code, could you please modify this part~
Thank you very much!

`preds_adv = np.array([])
preds_pur = np.array([])
ys = np.array([])
y_counter = 0

for i in range(1000):
    
  Adv, X, y = purified[i]
  X = X.to("cuda")
  Adv = Adv.to("cuda") 
  pred_pur = cnn(X)
  pred_adv = cnn(Adv)
  preds_adv = np.append(preds_adv,np.argmax(pred_adv.detach().cpu().numpy(),axis=1))
  preds_pur = np.append(preds_pur,np.argmax(pred_pur.detach().cpu().numpy(),axis=1))
  y_counter+=y.numpy()
  ys = np.append(ys, y.numpy())`

**Comment by cyc9805** [comment]
I also found the same error! It would be really nice of you to add complementary code about variable purifed.

**Comment by BalaMallikarjuna-G** [comment]
Adv, X, y = purified[i]

Could not find 'purified', exactly where it is created/updated?

**Comment by BalaMallikarjuna-G** [comment]
Adv, X, y = purified[i]

Could not find 'purified', exactly where it is created/updated?

**Comment by ankile** [comment]
Hello, everyone—I'm incredibly sorry for not seeing this before! I truly hope you still found the project useful. If you still have questions or wonder about this, please reach out! I'll monitor these issues better going forward, but otherwise my email larsankile@g.harvard.edu is a good place to reach out as well.

Best, Lars

**Comment by ankile** [comment]
I've now added back the missing line that @haloha123 first pointed out. The data that we're reading in there can be found in this [Google Drive folder](https://drive.google.com/drive/folders/1EfnbhnjbGE-rhIuBUhB8MmLXR8wK5yGL?usp=sharing). I hope this helps @haloha123, @cyc9805, @BalaMallikarjuna-G! 😊
